### PR TITLE
Many to many associations

### PR DESCRIPTION
**Pull request by ykostov**
WIP

### CODE REVIEW

File: lib/bgsite_official/categories/tag.ex [hunk 1]
```diff
@@ -17,4 +17,19 @@ defmodule BgsiteOfficial.Categories.Tag do
     |> cast(attrs, [:title, :banner, :description, :likes])
     |> validate_required([:title, :banner, :description, :likes])
   end
+
+  many_to_many(
+     :websites,
+     Website,
```
Comment (rmilushev) [inline lib/bgsite_official/categories/tag.ex:23]: The module name is : `defmodule BgsiteOfficial.Home.Websites do`

File: lib/bgsite_official/home/websites.ex [hunk 2]
```diff
@@ -20,6 +20,31 @@ defmodule BgsiteOfficial.Home.Websites do
     |> validate_required([:title, :description, :likes])
   end
 
+  many_to_many(
+    :tag,
+    Tag,
+    join_through: "tag_website",
```
Comment (rmilushev) [inline lib/bgsite_official/home/websites.ex:26]: The join table name is `website_tag`, please change it.

File: lib/bgsite_official/home/websites.ex [hunk 3]
```diff
@@ -20,6 +20,31 @@ defmodule BgsiteOfficial.Home.Websites do
...
+    websites =
+      Website
+      |> where([website], website.id in ^website_ids)
+      |> Repo.all()
+
+    with {:ok, _struct} <-
+           tag
+           |> Tag.changeset_update_website(websites)
+           |> Repo.update() do
+      {:ok, Accounts.get_tag(tag.id)}
```
Comment (rmilushev) [inline lib/bgsite_official/home/websites.ex:40]: IMO `Accounts` is not proper here, check, please.

**Review by rmilushev — CHANGES_REQUESTED** [review]
We will refactor the function for website -> tags association.

**Comment by ykostov** [comment]
New pull for this

**Pull request closed without merging**